### PR TITLE
Data alteration and theft

### PR DESCRIPTION
**Pull request by MalinaStefaniaStoicanescu**
Change-Id: Iebf91b95c649ca7b6171dce052ef9d38dd57b590
Signed-off-by: MalinaStefaniaStoicanescu <malina.stoicanescu@thalesgroup.com>

### CODE REVIEW

**Review by claudiasima-ro — APPROVED** [review]

File: plugins/org.polarsys.capella.cybersecurity.model/src/org/polarsys/capella/cybersecurity/model/activator/CybersecurityModelActivator.java [hunk 1]
```diff
@@ -44,6 +44,8 @@
   
   public static final String CYBERSECURITY_CFG_THREAD_KIND_KEYWORD = "Thread.Kind";
   public static final String THREAT_KIND_EAVESDROPPING_KEYWORD = "Eavesdropping";
+  public static final String THREAT_KIND_THEFT_KEYWORD = "Theft";
+  public static final String THREAT_KIND_DATA_ALTERATION_KEYWORD = "Data alteration";
   public static final String THREAT_KIND_THEFT_AND_DATA_ALTERATION_KEYWORD = "Theft and data alteration";
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.model/src/org/polarsys/capella/cybersecurity/model/activator/CybersecurityModelActivator.java:49]: Remove THREAT_KIND_THEFT_AND_DATA_ALTERATION_KEYWORD.

Comment (MalinaStefaniaStoicanescu) [reply plugins/org.polarsys.capella.cybersecurity.model/src/org/polarsys/capella/cybersecurity/model/activator/CybersecurityModelActivator.java]: done :)

File: plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki [hunk 2]
```diff
@@ -61,7 +64,7 @@ A situation that is unwanted by the stakeholders and that is to be avoided. Thre
 |}
 
 
-A ''Threat kinds'' can be customized by the user in the Cybersecurity Configuration, by editing the Threat.Kind type.
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki:64]: Why this change? No need.

**Review by georgiana-ecobici — APPROVED** [review]
